content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -202,6 +202,13 @@ example “`*.foo.com`”). Precise matches require that the HTTP `host` header
 matches the `host` field. Wildcard matches require the HTTP `host` header is
 equal to the suffix of the wildcard rule.
 
+
+With respect to Wildcards and Top-level-domains (eg. .com, .ca, .org, .edu), there is
+currently no support for specifying a wildcard in the format of `subdomain.domain.*`.
+This would assume that the domain owner has access to and purchased all 
+Top-level-domains for their domain in order for this to be supported.
```

Comment:
Not sure this is needed @distributethe6ix 

“Wildcard matches require the HTTP `host` header is equal to the suffix of the wildcard rule.” already covers this, I think.

Reply:
@distributethe6ix 

```suggestion
```

Remove this paragraph (accept the above suggestion) and we can merge this in?

Reply:
“Wildcard matches require the HTTP host header is equal to the suffix of the wildcard rule.”  does not satisfy what the original issue was asking for. 

With respect to Ingress, you cannot specify a Top level domain wild-card, hence why I added this bit into the documentation. It's not clear otherwise. 

The top-level-domain is the ".com, .ca, .org, .io" for example, is not part of the "host-field"

The wildcard functionality does not address situations where someone who has purchased multiple domains can route accordingly to .ca, .com. 

Happy to hop on a zoom/discord chat to further provide clarification.